COSQuickRef.md

```diff
@@ -31,110 +31,110 @@ A list of some common ObjectScript expressions
+| Date conversion (external → internal)            | `Set variable = $zdh("mm/dd/yyyy")`                         |
+| Date conversion (internal → external)            | `Set variable = $zd(internalDate, format)`                  |
+| Time conversion (external → internal)            | `Set variable = $zth("hh:mm:ss")`                           |
+| Time conversion (internal → external)            | `Set variable = $zt(internalTime, format)`                  |
+| Display current internal date/time string        | `Write $horolog`                                            |
+| Display UTC date/time string                     | `Write $ztimestamp`                                         |
 
 ## ObjectScript Branching Functions
 
-| Action                                           | Code                                                        | 
-|--------------------------------------------------|-------------------------------------------------------------| 
-| Display result for value of expression           | `Write $case(expression, value1:result1, value2:result2, …, :resultN)`                         | 
-| Display result for first true condition          | `Write $select(condition1:result1, condition2:result2, …, 1:resultN)`                  | 
+| Action                                           | Code                                                                   |
+|--------------------------------------------------|------------------------------------------------------------------------|
+| Display result for value of expression           | `Write $case(expression, value1:result1, value2:result2, …, :resultN)` |
+| Display result for first true condition          | `Write $select(condition1:result1, condition2:result2, …, 1:resultN)`  |
 
 ## ObjectScript String Functions
 
-| Action                                           | Code                                                        | 
-|--------------------------------------------------|-------------------------------------------------------------| 
-| Display substring extracted from string	                   | `Write $extract(string, start, end)`              | 
-| Display right-justified string within width characters     | `Write $justify(string, width)`                   |
-| Display length of string                                   | `Write $length(string)`                           |
-| Display number of delimited pieces in string               | `Write $length(string, delimiter)`                | 
-| Display piece from delimited string                        | `Write $piece(string, delimiter, pieceNumber)`    |
-| Set piece into delimited string	                           | `Set $piece(string, delimiter, pieceNumber) = piece` |
-| Display string after replacing substring                   | `Write $replace(string, subString, replaceString)`|
-| Display reversed string                                    | `Write	$reverse(string)`                          |
```

Comment:
This is important, thanks! Otherwise newbies will be much surprised to see it's not compiling well even after simple copy/paste. It should still be valid after copy-paste operation.